docs/src/config/ini-homing.adoc

```diff
@@ -6,10 +6,10 @@
 
 Homing sets the zero origin of the G53 machine coordinates. +
 Soft limits are defined relative to the machine origin. +
-The soft limits automatically decelerate and stop the axes before they hits the limits switches +
-A properly setup and functioning machine will not move beyond soft(ware) limits and +
+The soft limits automatically decelerate and stop the axes before they hit the limits switches +
+A properly configured and functioning machine will not move beyond soft(ware) limits and +
 will have the machine origin set as repeatable as the home switch/index mechanism is. +
-Linuxcnc can be  homed by eye (alignment marks), with switches, with switches and encoder index, or use absolute encoders. +
+Linuxcnc can be homed by eye (alignment marks), with switches, with switches and an encoder index, or by using absolute encoders. +
```

Comment:
IMHO, the original 'by using' make more sense or something like this.
Linuxcnc can be homed:
    by eye
   with switches
   with switches and encoder index
  or with absolute encoders

Reply:
The "by using" is the correction, not the original. Your itemized presentation is also fine.  Please amend.